File: 정렬/10989/10989.c
#include <stdio.h>
#define MAX 10001

int main()
{
   int n, i, j;
   int arr[MAX] = {0};
   scanf("%d", &n);
   for (i = 0; i < n; ++i)
   {
      scanf("%d", &j);
      ++arr[j];
   }

   for (i = 0; i < MAX; ++i)
      for (j = 0; j < arr[i]; ++j)
         printf("%d\n", i);   

   return 0;
}

File: 정렬/10989/10989.py
import sys
MAX = 10001

n = int(sys.stdin.readline())
arr = [0] * MAX
for i in range(n):
   temp = int(sys.stdin.readline())
   arr[temp] += 1

for i in range(MAX):
   for j in range(arr[i]):
      sys.stdout.write(str(i) + '\n')